server/db/db.js

```diff
@@ -84,10 +57,45 @@ async function execute(path, params = {}) {
   sql = sql.replace(/\$\{[^{}]+\}/g, queryParam);
   const values = queryVariables ? queryVariables.map(p => params[p]) : [];
   return query(sql, values);
-}
+};
+
+const migrate = async () => {
+  let existingMigrations = [];
+  try {
+    const result = await execute('server/sql/migrationQueries/get_all.sql');
```

Comment:
this file name (`migrationQueries`) is fine since we don't actually have a table for this, but generally for SQL db nomenclature, it's more common to use snake_case

i.e. the `user-payments` table would be live in the file structure as
```
server/sql/user_payments/whatever.sql
```

no need to change anything, just food for thought moving forward